### PR TITLE
Update apps_supported.adoc

### PR DESCRIPTION
**Pull request by jnweiger**
Starting with mediaplayer 1.0.4 gallery and mediaplayer can coexist

### CODE REVIEW

**Comment by mmattel** [comment]
In case 1.0.4 will be released with 10.7, things are easy as we can restrict the change to master and backport to 10.7 in docs.

If there is no 10.7 but a 10.6.x release of core, the content is not clear, because if backporting to 10.6, it would mean that all 10.6 releases are capable - which they are not. Then a different wording would be necessary to avoid misunderstandings/misbehaviour.

File: modules/admin_manual/pages/installation/apps_supported.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@
 * Files (cannot be disabled)
 * xref:installation/apps/mediaviewer/index.adoc[Files Media Viewer]
 +
-NOTE: The _Gallery_ and _Files VideoPlayer_ apps need to be uninstalled before installing the Media Viewer app.
+NOTE: The _Gallery_ and _Files VideoPlayer_ apps need to be disabled before using the Media Viewer app.
```
Comment (mmattel) [inline modules/admin_manual/pages/installation/apps_supported.adoc:20]:
```suggestion
NOTE: Before Files Media Viewer 1.0.4, the _Gallery_ and _Files VideoPlayer_ apps need to be **uninstalled before installing** the Media Viewer app. Starting with Files Media Viewer 1.0.4, the _Gallery_ and _Files VideoPlayer_ apps need to be **disabled before using** the Files Media Viewer app.
```

Comment (mmattel) [reply modules/admin_manual/pages/installation/apps_supported.adoc:20]: @jnweiger mind to take a look if you can live with that?

**Comment by JanAckermann** [comment]
@jnweiger @mmattel 
There has been a recent PR https://github.com/owncloud/core/pull/38132, which allows us to have multiple (default) apps present for a file. Wondering if all the apps can coexist and still enabled.

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
Backport to 10.5 and 10.6 needed

**Review by voroyam — APPROVED** [review]

**Comment by voroyam** [comment]
for some strange reasons the checks are failing. Created new PR here https://github.com/owncloud/docs/pull/3327

**Pull request closed without merging**